heareval/tasks/README.md

```diff
@@ -20,7 +20,7 @@ For each evaluation task, the directory structure is:
 
 ## More details
 
-task.json also specifies the hop_size that we will use for the
+task.json also specifies the frame_rate that we will use for the
```

Comment:
We still need to create this json file right? We're sticking with csv files for the labels, which are non-negative ints. And then this PR introduces a new csv file that maps integer labels to the original string labels, ya?

Reply:
We still need a file (task.json but could be something else) that specifies the task type (multiclass??) and frame_rate at the very minimum. If we don't want a README and LICENSE we could put URLs for them in here too